### PR TITLE
fix: explicitly set config instead of using changable default in tests

### LINKED ISSUE #4126: Workflow run 'Release' failed
@GreptimeTeam/db-approver
New failure: https://github.com/GreptimeTeam/greptimedb/actions/runs/9440573664

### PR DESCRIPTION
**Pull request by MichaelScofield**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## Refer to a related PR or issue link (optional)

fix #4126 

## What's changed and what's your intention?

fix the test failure in #4126 

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Does this mean that the default value is improper and we should revisit them later? Or just the one for this scenario need tunning.

File: src/cmd/tests/load_config_test.rs [hunk 1]
```diff
@@ -70,6 +71,11 @@ fn test_load_datanode_example_config() {
                 num_workers: 8,
                 auto_flush_interval: Duration::from_secs(3600),
                 scan_parallelism: 0,
```
Comment (evenyag) [inline src/cmd/tests/load_config_test.rs:73]: Also set `max_background_jobs` explicitly? We might adjust it later.

**Review by evenyag — APPROVED** [review]
LGTM

**Comment by evenyag** [comment]
> Does this mean that the default value is improper and we should revisit them later?

No. These default options are auto-tunable. They depend on the total memory and cores of the host machine running the test. We can't assert the value of the default option with a fixed value in the example config file.

**Pull request merged**